gadget-container/cleanup.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+# This script cleans up all the files installed by Inspektor Gadget
+
+for i in ocihookgadget runc-hook-prestart.sh runc-hook-poststop.sh ; do
+  echo "Removing $i..."
+  rm /host/opt/bin/$i
```

Comment:
Suggestion: `/bin/rm -f`
- full path to avoid a potential alias
- `-f` to avoid printing errors if the file is already removed

Reply:
Updated, thanks!